lnd_multi-hop_htlc_local_chain_claim_test.go

```diff
@@ -0,0 +1,352 @@
+// +build rpctest
```

Comment:
imo it'd be preferable to start moving these all into the `lntest` package, tho i suspect there are a bunch of dependencies either in `lnd` or just helper functions that would break if we don't do the move all at once. is that the case?

Reply:
It needs to be done all at once, commit added.